### ISSUE TITLE
Android only builds for armeabi-v7a, could other architectures be suported

### ISSUE THREAD

**Issue by NickoftheSouth** [halfnelson/moaicli#18]
Is there any way to include other architectures in Moaicli?

After running my app through an online Android device test server I found there was an unsatisfied linker error on a number of devices.
I'm specifically thinking of the orignal armeabi, but mips and x86 would also be useful.

**Comment by halfnelson** [comment]
yeah, 1.7.5 is probably your best bet. Give pito a go, it is almost up to moaicli
